scripts/run-integration-tests.sh

```diff
@@ -31,7 +31,7 @@ elif [ "$SUITE" = "react-percy" ]; then
   cd integration-tests/react-percy
   link react-percy
   link react-percy-webpack
-  yarn test
+  DEBUG="react-percy:*" yarn percy
```

Comment:
I need to add some basic logging about what `react-percy` is doing, but setting the debug env variable here will suffice for now.